spec/unit/azurerm_server_list_spec.rb

```diff
@@ -11,25 +11,25 @@
 
     @compute_client = double("ComputeManagementClient")
 
-    @server1 = double("server1", :name => "MyVM1", :id => double, :location => "west-us", :properties => double)
+    @server1 = double("server1", :name => "MyVM1", :id => double, :location => "west-us")#, :properties => double)

```

Comment:
Instead of commenting out, it would be good to delete `properties` part everywhere in the specs.